datafusion/core/src/physical_plan/sorts/merge.rs

```diff
@@ -52,7 +52,7 @@ macro_rules! merge_helper {
 }
 
 /// Perform a streaming merge of [`SendableRecordBatchStream`]
-pub(crate) fn streaming_merge(
+pub fn streaming_merge(
```

Comment:
For a public API, I suggest that we can provide more clear doc for it. E.g., this will sort stream based on provided expressions etc.

Reply:
Thanks, updated